kotlinx-coroutines-debug/src/DebugProbes.kt

```diff
@@ -59,13 +61,31 @@ public object DebugProbes {
      * thread is captured and attached to the coroutine.
      * This option can be useful during local debug sessions, but is recommended
      * to be disabled in production environments to avoid stack trace dumping overhead.
+     *
+     * `true` by default.
      */
     public var enableCreationStackTraces: Boolean
         get() = DebugProbesImpl.enableCreationStackTraces
         set(value) {
             DebugProbesImpl.enableCreationStackTraces = value
         }
 
+    /**
+     * Whether to ignore coroutines that have their context equal to [EmptyCoroutineContext].
```

Comment:
```suggestion
     * Whether to ignore coroutines whose context is [EmptyCoroutineContext].
```
Reflects the code better, if nothing else.